Package.swift

```diff
@@ -22,7 +22,7 @@ let package = Package(
         // Targets are the basic building blocks of a package. A target can define a module or a test suite.
         // Targets can depend on other targets in this package, and on products in packages which this package depends on.
         .target(
-            name: "Mint",
+            name: "mint",
```

Comment:
Instead of changing the target name to lowercase, I'd rather specify an explicit product with a lowercase name
```
.executable(name: "mint", targets: ["Mint"]),
```

Reply:
@yonaskolb, I will double check, but I am pretty sure that the `name` argument for products is only used in the manifests of other packages where they specify which of your products to actually use. I think it is the target name that defines the name of the executable file itself, and hence what you use from the shell.

If that is correct, then it would be reasonable to make the executable name uppercase if you want (for referencing in manifests), but **the target name needs to lowercase unless you want the command to always be used capitalized, i.e. `$ Mint install ...`**, which would be a little unusual.

Reply:
Just tested it. My assumption was wrong. Holdover from Swift 3 I guess.

Reply:
I will fix it.

Reply:
Can you rename the `mint` folder back to `Mint` as well then

Reply:
Actually, it only sort of works. The executable file gets the product name, but when the executable file differs from the corresponding target name (which I imagine is rare) some of `swift package`’s subcommands start to have trouble. So I would still highly recommend using `mint` for both the target and product (if you want to start exposing it as a product).

Reply:
Interesting. What sort of subcommands have trouble?

Reply:
`describe`, `dump-package`, etc.